SU-SOE/stanford_soe_helper
# Collections Injector file 37 into SCSS

# READY FOR REVIEW

# Summary
This adds the Collections Injector file 37 into SCSS.

# Needed By (Date)
- Sprint end

# Criticality
- Collections Injector file 37 into SCSS


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-2919 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2919)
- delete this injector: `admin/config/development/css-injector/edit/37`
- `drush cc all`
- go to: `magazine/future-everything` and make sure the css is showing.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2919

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
